### DISCUSSION TITLE
Folder function, containing several albums for sharing (or not)

### DISCUSSION THREAD

**Discussion by djibi400** [immich-app/immich#10746] [General]
Hello everybody!

I own the much-missed [Monument 2](https://www.getmonument.com/m2/monument-2-photo-storage-device) (which has been, as it seems, almost abandoned for now), with which I enjoyed a great feature they offered and which I'd love to find in Immich:

Would it be possible to have a folder function in which Immich albums could be added/created, and that this folder could then be shared the same way an album is shared, giving access to all the albums that folder contains?

Let me explain to you my real-life situation:
In Immich, I use it every day to back up and manage my personal photos (that I keep private), which could be (optionally) stored in **folder A**

Then, I have a lot of old family photos, already arranged in many albums, that would be added into **folder B** that I would like to share (or add user access) to a family member

The family member can then access this link whenever he or she wants, and if I modify, add, or delete albums in this folder, the link would display the updated content in real time

![Artboard 1](https://github.com/immich-app/immich/assets/13452716/083d7b53-dc94-4fb4-b172-e566f9acefdd)

Let me know if you think this functionality is relevant; it would be a key feature for me :)

Thank you for this amazing piece of software, you're doing a hell of a job.
++

**Comment by bo0tzz** [comment]
#2073, #5078, #5409
